src/node_dotenv.cc

```diff
@@ -4,7 +4,9 @@
 #include "uv.h"
 
 namespace node {
-
```

Comment:
This line removal seems to be unnecessary.

Reply:
Again, I have a c++ formatter (That obeys the existing linter rules). I think one of the linter rules is no extra spaces.